app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -778,6 +787,7 @@ public function __construct(
         Product\TaxClassProcessor $taxClassProcessor,
         \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
         \Magento\Catalog\Model\Product\Url $productUrl,
+        StockItemImporterInterface $stockItemImporter,
```

Comment:
currently adding required dependency will introduce Backward Incompatible changes.

It's better to do like we usually do to preserve constructor BC : 
```
 public function __construct(
        \\old arguments 
        StockItemImporterInterface $stockItemImporter = null //add new dependency at the end of the dependencies list
    ) {
        
        $this->stockItemImporter = $stockItemImporter ?: \Magento\Framework\App\ObjectManager::getInstance()->get(
            StockItemImporterInterface::class
        );
```

Reply:
@maghamed Understood - i've just pushed up a commit to fix this [see here](https://github.com/magento-engcom/msi/pull/211/commits/1af449357e08c0656ce825286d587f99bb17f020)